cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/azure/wizard-basic/management-credential-step/login/Login.tsx

```diff
@@ -0,0 +1,132 @@
+// React imports
+import React, { ChangeEvent, useContext } from 'react';
+
+// Library imports
+import { CdsInput } from '@cds/react/input';
+import { CdsIcon } from '@cds/react/icon';
+import { UseFormReturn } from 'react-hook-form';
+import { CdsControlMessage } from '@cds/react/forms';
+import { CdsSelect } from '@cds/react/select';
+
+// App imports
+import { FormField, FormInputs } from '../ManagementCredentials';
+import { STORE_SECTION_FORM } from '../../../../../../state-management/reducers/Form.reducer';
+import { AzureStore } from '../../../../../../state-management/stores/Azure.store';
+import { AzureClouds } from '../../../../../../shared/constants/App.constants';
+import { CdsButton } from '@cds/react/button';
+
+interface Props {
+    connected: boolean;
+    handleInputChange: (field: FormField, value: string) => void;
+    handleConnect: () => void;
+    methods: UseFormReturn<FormInputs, any>;
+}
+
+export default function Login(props: Props) {
+    const { azureState } = useContext(AzureStore);
+    const {
+        methods: {
+            formState: { errors },
+            register,
+        },
+        connected,
+        handleInputChange,
+        handleConnect,
+    } = props;
+
+    const dataEntered = () => {
+        const fields = ['SUBSCRIPTION_ID', 'TENANT_ID', 'CLIENT_ID', 'CLIENT_SECRET', 'AZURE_ENVIRONMENT'];
```

Comment:
I think it'd be nicer to use an enumeration of these fields, and then refer to them below when returning the DOM

Reply:
fixed.